### PR TITLE
added travis CI support

### PR DESCRIPTION
**Pull request by mvgaliev**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  - linux
+sudo: required
+dist: trusty
+language: node_js
+node_js:
+  - "7"
+install:
+  - npm install
+script:
+  - npm run lint
```
Comment (ignatvilesov) [inline .travis.yml]: Please include ```npm run package``` as well.

File: pbiviz.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "supportUrl": "http://community.powerbi.com/",
     "gitHubUrl": "https://github.com/microsoft/PowerBI-visuals-decision-tree"
   },
-  "apiVersion": "1.2.0",
+  "apiVersion": "1.6.0",
```
Comment (ignatvilesov) [inline pbiviz.json]: Please do a quick check of the visual with API 1.6.0.

File: package.json [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+  "scripts": {
+    "postinstall": "pbiviz update 1.6.0",
+    "pbiviz": "pbiviz",
+    "package": "pbiviz package",
+    "lint": "tslint -r \"node_modules/tslint-microsoft-contrib\" \"+(src|test)/**/*.ts\""
+  },  
+  "devDependencies": {
+    "tslint": "^4.4.2",
+    "tslint-microsoft-contrib": "^4.0.0",
+    "typescript": "2.1.4",
```
Comment (ignatvilesov) [inline package.json]: There's no reason to include the TS. Please remove it.

File: .travis.yml (hunk 1 above)
Comment (mvgaliev) [inline .travis.yml]: Added

File: pbiviz.json (hunk 2 above)
Comment (mvgaliev) [inline pbiviz.json]: Quick check result is ok

**Pull request merged**